resources/definitions/fdmprinter.def.json

```diff
@@ -3379,6 +3379,30 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "support_skip_some_zags":
+                {
+                    "label": "Skip Some ZigZags Connections",
+                    "description": "Skip some ZigZags connections to make the support structure easier to break.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "support_enable and (support_pattern == 'zigzag')",
+                    "limit_to_extruder": "support_infill_extruder_nr",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": true
+                },
+                "support_zag_skip_count":
```

Comment:
Maybe it's better to make a parent setting `support_zag_skip_frequency` in 'mm^-1'.
The average amount of connections to be skipped per mm.

Breaking away support depends more on the size of the support chunk than on the amount of not-removed connections.
When changing the support infill density it would be nice to automatically also change the support_zag_skip_count by its inheritance function.

support_zag_skin_count could then have an inheritence function like `max(1, round(1 / support_zag_skip_frequency / support_line_distance))`